src/sentry/utils/snuba.py

```diff
@@ -369,7 +369,10 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
             referrer=referrer, is_grouprelease=is_grouprelease, totals=totals, limitby=limitby
         )
     except (QueryOutsideRetentionError, QueryOutsideGroupActivityError):
-        return OrderedDict()
+        if totals:
+            return OrderedDict(), {}
```

Comment:
`{}` isn't perfect, because a real query with no results would still return a `totals` dict with proper keys (all set to their 0 value)... creating a "fake" zero-ed out dict didn't seem worth it? Should be up to the caller to check for valid results or `totals.get` I think?

Reply:
Yeah I'm fine with that, I guess you fixed the one place that blindly tried to grab stuff from totals.